openlibrary/templates/account/create.html

```diff
@@ -28,12 +28,16 @@ <h1>$_("Sign Up")</h1>
                 <span class="smaller lighter">$input.help</span>
         </div>
         <div class="input">
+            <script src="https://kit.fontawesome.com/a076d05399.js"></script>
             $:input.render()
+            $if input.name == 'password':
+            $:suffix
```

Comment:
```suggestion
                <span class="password-visibility-toggle password-visibility-toggle--hidden"></span>
```
There is no `suffix` being passed for the password field.  Just render the icon here, instead.